samples/Azure.ResourceManager.Sample/Generated/Models/AvailabilitySets.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't think these belong in the Model folder, they should be in the same level as the [Resource]Operations classes